tools/test/executable_benchmarks.mlir

```diff
@@ -0,0 +1,19 @@
+// RUN: iree-compile %s -o ignored.mlir \
+// RUN:     --iree-hal-target-backends=vmvx \
+// RUN:     --iree-hal-dump-executable-benchmarks-to=- | \
+// RUN: iree-compile - | \
+// RUN: iree-benchmark-module | \
```

Comment:
This test probably runs quickly enough, but we could pass a "max iterations" flag

Reply:
yeah, completes in ~10ms - since this is an example to point people at I'd rather keep it terse to avoid confusion

Reply:
10ms is great :) I'd be concerned if it took > 1 minute (based on the whims of the benchmark framework)